lib/executables_db.rb

```diff
@@ -54,40 +56,45 @@ def changed?
     def update!(update_existing: false, install_missing: false, max_downloads: nil)
       downloads = 0
       disabled_formulae = []
-      Formula.all.each do |f|
-        break if max_downloads.present? && downloads > max_downloads.to_i
-        next if f.tap?
 
-        name = f.full_name
+      Tap.each do |tap|
```

Comment:
@woodruffw Probably want to limit this to official taps only without `--eval-all` or just require `--eval-all` or `HOMEBREW_EVAL_ALL` to be passed to the relevant commands?

For context: the concern here is arbitrary Ruby execution from untrusted taps/formulae.

Reply:
Yeah, good call -- I'll have some time to update here in the coming days 🙂

Reply:
@woodruffw Gentle nudge

Reply:
Thanks for the nudge, looking again today 😅

Reply:
`brew which-update` now takes `--eval-all`, which is disabled by default (meaning that, by default, only `CoreTap.instance` is used).